var/.htaccess

```diff
@@ -1,8 +0,0 @@
-<IfVersion < 2.4>
```

Comment:
Don't remove .htaccess in /var directory. If you do so, var directory is removed, so other developers will have to create it. Please move these lines back